src/chainstate/burn/db/sortdb.rs

```diff
@@ -2742,29 +2744,27 @@ impl SortitionDB {
         Ok(Some((snapshot, transition_ops)))
     }
 
+    /// Evaluate the sortition (SIP-001 miner block election) in the burnchain block defined by
+    /// `burn_header`. Returns the new snapshot and burnchain state
+    /// transition.
     ///
     /// # Arguments
     /// * `burn_header` - the burnchain block header to process sortition for
     /// * `ops` - the parsed blockstack operations (will be validated in this function)
     /// * `burnchain` - a reference to the burnchain information struct
     /// * `from_tip` - tip of the "sortition chain" that is being built on
     /// * `next_pox_info` - iff this sortition is the first block in a reward cycle, this should be Some
-    ///
-    pub fn evaluate_sortition(
+    /// * `announce_to` - a function that will be invoked with the calculated reward set before this method
+    ///                   commits its results.
```

Comment:
Can you add to this comment to describe its concrete use-case?  "For example, this function is used to post the calculated reward set to an event observer" or something like that.

Reply:
Expanded in 8608cc7fd